src/start.sh

```diff
@@ -0,0 +1,4 @@
+./target/release/el_monitorro &
```

Comment:
I think we shouldn't start all binaries in the same container. Can we modify the entrypoint so binary could be selected dynamically?

Reply:
why ? its perfectly working for me.

so can we tell me the correction ? because i didnt get that